src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/project.json

```diff
@@ -21,31 +21,36 @@
   },
 
   "dependencies": {
-    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.1, 4.0)"
+    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.5, 4.0)"
   },
 
   "frameworks": {
     "net45": {
+      "buildOptions": {
+        "define": [ "NET45" ],
+        "debugType": "portable"
+      },
       "dependencies": {
       }
     },
     "netstandard1.5": {
+      "buildOptions": {
+        "define": [ "NETSTANDARD11", "PORTABLE" ],
```

Comment:
@hovsepm do we need net standard 1.1 monikers?
Also we should completely remove all "portable" monikers. The term portable is confusing and not sure what exactly it means.